examples/eks/cassandra-stress-write.yaml

```diff
@@ -0,0 +1,295 @@
+apiVersion: batch/v1
```

Comment:
This is generated by https://github.com/scylladb/scylla-operator/blob/master/scripts/cass-stress-gen.py, maybe we should add a stress testing example as a separate thing.

Reply:
removed